opengrok-indexer/src/main/java/org/opengrok/indexer/util/ClassUtil.java

```diff
@@ -183,30 +189,29 @@ private static Method getSetter(Object obj, String fieldName) throws IOException
      *
      * @param obj the object
      * @param fieldName name of the field which will be changed
-     * @param value desired value
+     * @param value desired value represented as string
      *
      * @throws IOException if any error occurs (no suitable method, bad conversion, ...)
      */
     public static void invokeSetter(Object obj, String fieldName, String value) throws IOException {
         Method setter = getSetter(obj, fieldName);
         Class<?> c = setter.getParameterTypes()[0];
-        Object objValue = getObjectValue(c, value);
+        Object objValue = getObjectValue(fieldName, c, value);
         // TODO this will get the setter again => refactor
         invokeSetter(obj, fieldName, objValue);
     }
 
+    /**
+     *
+     * @param obj the object
+     * @param fieldName name of the field which will be changed
+     * @param value desired value
+     * @throws IOException
```

Comment:
maybe add a note that all exceptions from the reflection are propagated as ioexception